README.md

```diff
@@ -226,20 +230,37 @@ python runner.py --wf BTA_addAllTracks --json metadata/test_bta_run3.json --camp
 </p>
 </details>
 
-## Scale-out (Sites)
+## Scale-out 
 
 Scale out can be notoriously tricky between different sites. Coffea's integration of `slurm` and `dask`
 makes this quite a bit easier and for some sites the ``native'' implementation is sufficient, e.g Condor@DESY.
-However, some sites have certain restrictions for various reasons, in particular Condor @CERN and @FNAL.
+However, some sites have certain restrictions for various reasons, in particular Condor @CERN and @FNAL. The scaleout scheme named with `$cluster_schedule_system/scheduler/site`. The existing sites are documented in [sites configuration](#sites-configuration-with-daskparsl-schedular) while [standalone condor submission](#standalone-condor-jobslxpluscmsconnect) are possible and strongly suggest used in lxplus. 
```

Comment:
The scaleout scheme named with -> The scaleout scheme is named as follows:

are possible and strongly suggest used in lxplus. -> is possible and strongly suggested when working on lxplus.